crates/flz/Cargo.toml

```diff
@@ -15,8 +15,14 @@ exclude.workspace = true
 workspace = true
 
 [dependencies]
+fastlz = { version = "0.1.0", optional = true }
 
 [dev-dependencies]
 revm.workspace = true
 rstest.workspace = true
 alloy-sol-types.workspace = true
+
+[features]
+default = []
```

Comment:
sure neither `compress` nor `decompress` should be default features or both?

Reply:
In the future, if we find that we _do_ end up using the re-exports, we can always change this.